packages/renderer/src/lib/container/ContainerDetailsTerminal.spec.ts

```diff
@@ -39,13 +39,19 @@ vi.mock('xterm', () => {
   };
 });
 
+const matchMediaMock = vi.fn();
```

Comment:
why do we need to define matchMediaMock ?

Reply:
Fixed by e7f3ec1